cmd/rookctl/rook/rook.go

```diff
@@ -52,15 +50,13 @@ https://github.com/rook/rook`,
 }
 
 func init() {
-	defaultHost := os.Getenv("ROOK_API_SERVICE_HOST")
-	if defaultHost == "" {
-		defaultHost = "127.0.0.1"
+	host := os.Getenv("ROOK_API_SERVICE_HOST")
+	port := os.Getenv("ROOK_API_SERVICE_PORT")
+	defaultEndpoint := fmt.Sprintf("%s:%s", host, port)
+	if host == "" || port == "" {
+		fmt.Println("$ROOK_API_SERVICE_HOST or $ROOK_API_SERVICE_PORT are not set. Check if the rook-api service is running.")
```

Comment:
i think this change to notify the user that the endpoint to the API service is not known, instead of being silent and timing out each command.  I think the old default of 127.0.0.1 is legacy from standalone.